### ISSUE TITLE
Region change not working.

### ISSUE THREAD

**Issue by BOBdotEXE** [Plailect/Guide#63]
This is like the 4th guide I've tried, I followed this guide to setup a9lh, but I can't seem to change the region on my emunand.
 
The step I'm stuck on is where you need to downgrade with the other region files.
 
The downgrade runs with no errors, but it reboots back to the same software version I was on before.
 
I've tried just about every emunand backup I have 9.2, 10.3, 10.7, etc, all have the exact same issue.
 
I've run sysupdater about 3-5 times on each version, (even tried restoring in between flashes, and formatting [making sure to stay unlinked])
 
But nothing seems to work.

**Comment by Plailect** [comment]
To be clear, you're installing 9.2 region change on 10.7 and staying on 10.7?

**Comment by BOBdotEXE** [comment]
Thanks for taking the time to reply.
and my first attempt was with a 9.2emu region change, but that gave the issue I described above,
so I tried my other nand backups as well. (with later firmwares)
 
Eventually, I'd like to be on 10.7J, but for now I'd settle for just switching to at least 9.2J as that's what all the guides are for, and I can always update later,

**Comment by Plailect** [comment]
You can update after switching.

New or old 3DS?

**Comment by BOBdotEXE** [comment]
Ok, I just got back from updating, I think the issue was I was on a fraken firmware (copied from sysnand),
After updating, to 10.7 downgrading, It rebooted into USA, but I was unable to launch any apps (it said: "you need to upgrade to run this" on everything including setting!"
 
(rebooting did not help)
 
so I just injected the 'SecureInfo_A' and now everything seems to magically be working!
 
Let me just (first backup temunand, then) try an update real quick to make sure there are no issues.
 
[o3ds XL]

**Comment by Plailect** [comment]
Yay! :D

**Comment by BOBdotEXE** [comment]
While this is going, Quick question. the  'SecureInfo_A' is included in standard backups and restores right?
 
I don't want to accidentally restore my other emunand (usa) to a partition with a JP  'SecureInfo_A'.

**Comment by Plailect** [comment]
It is.

**Comment by BOBdotEXE** [comment]
Ok, thanks! 
 
And thanks for the great guide! I wish that was out when I did my initial downgrade! 
 
The whole process took be like 6 hours, and I ran into a lot of problems,
 
But your guide for going to CFW to a9lh Was great! no issues, and even with the extra steps needed I completed it in a fraction of the time!
 
---
 
Anyway, the update was successful! That means I'm good to go!
 
I just need to clean up the USA system titles that were left over.
 
Thanks again for taking the time to help me! :)
